eslint-plugin-prettier.js

```diff
@@ -0,0 +1,306 @@
+/**
+ * @fileoverview Runs `prettier` as an ESLint rule.
+ * @author Andres Suarez
+ */
+
+'use strict';
+
+// ------------------------------------------------------------------------------
+//  Requirements
+// ------------------------------------------------------------------------------
+
+const diff = require('fast-diff');
+const docblock = require('jest-docblock');
+
+// ------------------------------------------------------------------------------
+//  Constants
+// ------------------------------------------------------------------------------
+
+// Preferred Facebook style.
+const FB_PRETTIER_OPTIONS = {
+  singleQuote: true,
+  trailingComma: 'all',
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  parser: 'flow'
+};
+
+const LINE_ENDING_RE = /\r\n|[\r\n\u2028\u2029]/g;
```

Comment:
Can this be a non-global regex? It seems like it's only used as `LINE_ENDING_RE.test(something)`.